### PR TITLE
feat: added responsives to library

### PR DESCRIPTION
**Pull request by KanekarSnehal**
Added reponsiveness to library. Please review.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -69,9 +68,9 @@ a:hover {
     }
 
     .hero-img {
-        padding: 0 2.5rem 1rem;
+        /* padding: 0 2.5rem 1rem; */
```
Comment (amarphule) [inline style.css:71]: remove comment

File: index.html [hunk 2]
```diff
@@ -13,39 +13,36 @@
...
-                    <a href="/docs/documentation.html">
-                        <button class="btn primary-btn py-8">Get Started</button>
-                    </a>
-                
-            </div>
-           
-                <img class="hero-img" src="/assets/Innovation.gif" alt="hero animation">
-           
-        </main>
+    <main class="main-container" style="overflow-y: hidden;">
```
Comment (amarphule) [inline index.html:30]: avoid inline style

File: docs/docs-components/components/navigation.html [hunk 3]
```diff
@@ -13,6 +13,30 @@
 
 <body>
     <div class="documentation-page">
+        <div class="mobile-nav px-32">
+            <i class="fas fa-times close-btn position-absolute">
+            </i>
+            <ul class="list-style-none  my-32">
+            <li class="py-8"><a href="/index.html">Home</a></li>
+            <li><a href="">Installation</a></li>
```
Comment (nerdyspook) [inline docs/docs-components/components/navigation.html:21]: You should keep `the href="#"` or you can remove the `<a>` if you don't want it to redirect anywhere.

**Review by amarphule — APPROVED** [review]
Great work, Check given some suggestions

**Review by tewarimohit — APPROVED** [review]
Looks good to merge..... Godspeed..

**Review by nerdyspook — APPROVED** [review]

File: docs/docs-components/components/navigation.html (hunk 3 above)
Comment (KanekarSnehal) [reply docs/docs-components/components/navigation.html]: Made the changes please check.

File: style.css (hunk 1 above)
Comment (KanekarSnehal) [reply style.css]: done

File: index.html (hunk 2 above)
Comment (KanekarSnehal) [reply index.html]: Made the changes.

**Comment by KanekarSnehal** [comment]
> Looks good to merge..... Godspeed..

Thank you Mohit.

**Comment by KanekarSnehal** [comment]
Made the suggested changes, merging this PR.

**Pull request merged**